plugin/src/main/java/com/playmonumenta/epicstructures/managers/RespawnManager.java

```diff
@@ -39,14 +46,23 @@ public void run()
 	private boolean taskScheduled = false;
 	private boolean structuresLoaded = false;
 
-	public final String mZoneLayerName = "Respawning Structures";
-	protected ZoneLayer<RespawningStructure> mZoneLayer = new ZoneLayer<RespawningStructure>(mZoneLayerName);
-	public BaseZoneTree<RespawningStructure> mZoneTree;
+	public final String mZoneLayerNameInside = "Respawning Structures Inside";
+	public final String mZoneLayerNameNearby = "Respawning Structures Nearby";
+	protected ZoneLayer mZoneLayerInside = new ZoneLayer(mZoneLayerNameInside);
+	protected ZoneLayer mZoneLayerNearby = new ZoneLayer(mZoneLayerNameNearby, true);
+	protected Map<Zone, RespawningStructure> mStructuresByZone = new LinkedHashMap<Zone, RespawningStructure>();
```

Comment:
How does this work? This is never written to.
Edit: Ah, I see - it's accessed via a different class and written to. This is weird IMO - I think we should make it private and add a method to this class to put stuff into this map.